pbs/usersync.go

```diff
@@ -77,6 +78,7 @@ type UserSyncDeps struct {
 	RecaptchaSecret    string
 	HostCookieSettings *HostCookieSettings
 	Metrics            metrics.Registry
+	Analytics          *analytics.PBSAnalyticsModule
```

Comment:
People don't really use interface pointers in Go.

Design idioms are:

- No variables, functions, or anything else should declare "pointer to interface" as a type.
- Make _the pointer_ implement the interface by declaring `func (t *MyStruct) InterfaceImp()` functions.
- Pass `&MyStruct` into functions which use the interface.